test/recipes/70-test_quic_multistream.t

```diff
@@ -16,6 +16,9 @@ setup("test_quic_multistream");
 plan skip_all => "QUIC protocol is not supported by this OpenSSL build"
     if disabled('quic');
 
+plan skip_all => "QUIC multithreading is not supported by OpenSSL PUT builds"
+    if ! disabled('_PUT_MODEL_');
+
```

Comment:
This is not right. There is no such thing as `disabled('_PUT_MODEL_')` so this always evaluates to false -> the test is always disabled on all platforms.

Reply:
I assume you could call TEST_skip() in quic_multistream_test.c in setup_tests() based on the `_PUT_MODEL_` define or if that won't work, that you could call it in the individual test routines (test_dyn_frame_types, test_script).

Reply:
Not sure why do you think that `disabled('<define>')` should work. The `disabled('something')` function is for checking the Configure disable/enable options such as ec, quic, fips, etc.
Please see `util/perl/OpenSSL/Test/Utils.pm`

Reply:
Weird that it worked. The define is specified in 50-nonstop.cnf. What other option?

Reply:
It did not really work. As I wrote above the condition as you have it skips the test everywhere which means it of course also fixes the failure on PUT_MODEL.

And as I wrote above this could be handled by skipping in the C source file for the test instead of the test recipe.

Reply:
Change made, please re-review.